src/app/app.component.ts

```diff
@@ -1,46 +1,39 @@
-import { Component, ViewChild } from '@angular/core';
-import { Nav, Platform } from 'ionic-angular';
+import { Component } from '@angular/core';
+import { Platform } from 'ionic-angular';
 import { StatusBar } from '@ionic-native/status-bar';
 import { SplashScreen } from '@ionic-native/splash-screen';
 
-import { WiFiCallPage } from '../pages/wificall/wificall';
+import { TabsPage } from '../pages/tabs/tabs';
 import { AdminPage } from '../pages/admin/admin';
-
-type page = {title: string, component: any};
+import { Store } from '@ngrx/store';
+import { AppState } from '../store';
 
 @Component({
-  templateUrl: 'app.html'
+  template: `
+    <main [state]="store | async" [pages]="pages"></main>
+  `
 })
 export class WiFiCalling {
-  @ViewChild(Nav) nav: Nav;
-  rootPage: any = WiFiCallPage;
-  pages: Array<page>;
-  activePage: page;
 
-  constructor(platform: Platform, statusBar: StatusBar, splashScreen: SplashScreen) {
+  pages;
+
+  constructor(platform: Platform, statusBar: StatusBar, splashScreen: SplashScreen, public store: Store<AppState>) {
+
     platform.ready().then(() => {
       statusBar.styleDefault();
       splashScreen.hide();
     });
 
     this.pages = [
-      {title: 'WifiCall', component: WiFiCallPage},
+      {title: 'WifiCall', component: TabsPage},
+      {title: 'Login', component: 'AccessPage'},
```

Comment:
Because it is a lazy load page, we use the page name as string such as `this.navCtrl.push('OptionsPage');`